.gitignore

```diff
@@ -1,4 +1,3 @@
 coverage
 node_modules
 npm-debug.log
-package-lock.json
```

Comment:
If you can, please revert this part of the change and remove the `package-lock.json`. The issue is that when you install a module, the `package-lock.json` is not honored (https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json "The difference is that package-lock.json is that it cannot be published, and it will be ignored if found in any place other than the root project.") which means the versions that we test locally and CI will not be the same as what users get when they install it.

Reply:
@dougwilson done as you said